src/WebJobs.Script/Eventing/ScriptEventManager.cs

```diff
@@ -11,7 +11,13 @@ public sealed class ScriptEventManager : IScriptEventManager, IDisposable
         private readonly Subject<ScriptEvent> _subject = new Subject<ScriptEvent>();
         private bool _disposed = false;
 
-        public void Publish(ScriptEvent scriptEvent) => _subject.OnNext(scriptEvent);
+        public void Publish(ScriptEvent scriptEvent)
+        {
+            if (!_disposed)
```

Comment:
I assume these aren't really an issue since the host is shutting down anyway? (same with the BlobLease one below)